bin/serverless

```diff
@@ -2,7 +2,18 @@
 
 'use strict';
 
-let argv = require('minimist')(process.argv.slice(2));
+let argv = require('minimist')(process.argv.slice(2)),
```

Comment:
@erikerikson Is there any way we can move this logic into `Serverless.js`?  We're trying to keep this file lean.